### ISSUE TITLE
support router5 7.0

### ISSUE THREAD

**Issue by pzmosquito** [LeonardoGentile/mobx-router5#16]

**Comment by Mousecop** [comment]
Hello, I am also curious when an update will be available for this? Thank you.

**Comment by LeonardoGentile** [comment]
Hi, several things have changed in router5@7 and I've asked the creator some details on removed code that would affect this plugin: https://github.com/router5/router5/issues/390

I'm awaiting a reply

**Comment by pzmosquito** [comment]
Im trying to understand the question, which plugin are you trying to check in router5?

**Comment by LeonardoGentile** [comment]
@pzmosquito you are right, I usually carry out development in tandem with https://github.com/LeonardoGentile/react-mobx-router5

And this check is required from react-mobx-router5 that wouldn't work without mobx-router5 (this plugin) installed

**Comment by pzmosquito** [comment]
Can you do a `try... catch...` to make sure `mobx-router5` is in place?

**Comment by gimtonic** [comment]
Please fix your readme.md and write version router5 when work...  
 
Which version router5 can I use??? version 6 work??? Thanks.  
 
```json
"react-router5": "^7.1.0",
"router5": "^7.0.2",
"router5-plugin-browser": "^7.0.2",
"router5-plugin-logger": "^7.0.2" 
"mobx": "^5.15.1",
"mobx-react": "^6.1.4",
"mobx-router5": "^4.3.0",
```
